### PR TITLE
[Injimob 2265] show verifier name in VP sharing screen and consent screen and add translations for same

### PR DESCRIPTION
**Pull request by PuBHARGAVI**
## Description

> Changed the verifier title in VP sharing screen from **Requester** to **Verification Entity** and made changes to show the received verifier's name in consent screen.
> Added translations for the same in all languages.

## Issue ticket number and link

> [INJIMOB-2265](https://mosip.atlassian.net/browse/INJIMOB-2265)

[INJIMOB-2265]: https://mosip.atlassian.net/browse/INJIMOB-2265?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Alka1703 — APPROVED** [review]
LGTM

File: ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -34,7 +34,7 @@
       "location" : "https://github.com/mosip/inji-openid4vp-ios-swift",
       "state" : {
         "branch" : "develop",
-        "revision" : "7529cd4b0342f8d5c98c37f7ed3961b7073bf730"
+        "revision" : "6473093e1494fe240b77f07a5fcb0164a8e70c55"
```
Comment (Alka1703) [inline ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved:37]: why is there a change in this file? There is no ios side changes

Comment (PuBHARGAVI) [reply ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved]: @Alka1703 We made changes on the openid4vp swift library as well so update the commit with the latest one

File: machines/openID4VP/openID4VPSelectors.ts [hunk 2]
```diff
@@ -106,7 +106,10 @@ export function selectIsFaceVerifiedInVPSharing(state: State) {
 }
 
 export function selectVerifierNameInVPSharing(state: State) {
-  return state.context.authenticationResponse['client_metadata']?.name;
+  return (
+    state.context.authenticationResponse['client_metadata']?.['client_name'] ??
```
[the remark this answers is not in the document]
Comment (PuBHARGAVI) [reply machines/openID4VP/openID4VPSelectors.ts:110]: yes sreenadh
it is mentioned in the spec -> https://www.rfc-editor.org/rfc/rfc7591.txt
please refer to page 9

[the remark this answers is not in the document]
Comment (sree96) [reply machines/openID4VP/openID4VPSelectors.ts:110]: https://www.rfc-editor.org/rfc/rfc7591#section-2

**Review by sree96 — APPROVED** [review]

**Pull request merged**